modules/oauth-api/src/main/java/org/mule/runtime/oauth/api/builder/OAuthDancerBuilder.java

```diff
@@ -29,6 +29,15 @@
    */
   OAuthDancerBuilder<D> clientCredentials(String clientId, String clientSecret);
 
+  /**
+   * Define where the client credentials will be sent in the OAuth2 Dance.
```

Comment:
will be sent in the OAuth2 Dance -> will be sent in the HTTP requests for the OAuth2 Dance